### PR TITLE
add render time in viewer

### PR DESCRIPTION
**Pull request by JulianKnodt**
Adds a slider for render time in the viewer.

Only displays when the model has temporal distortion.

Is "render time" self-explanatory or is other wording better?

### CODE REVIEW

**Comment by JulianKnodt** [comment]
image

New slider, is getting piped properly into the server, now need to figure out when to display/pipe data from the server. Not sure where to read the graph information.

File: nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx [hunk 1]
```diff
@@ -164,6 +197,17 @@ export function RenderControls() {
         '1024px': () => setControls({ max_resolution: 1024 }),
         '2048px': () => setControls({ max_resolution: 2048 }),
       }),
+      // Dynamic NeRF rendering time
+      ...(display_render_time ? {render_time: {
+        label: 'Render Time',
```
Comment (tancik) [inline nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx:202]: Maybe `Render Timestep`

**Review by tancik — COMMENTED** [review]
When changing or adding messages between the server and client we also need to update the version number - https://github.com/nerfstudio-project/nerfstudio/blob/3d70b4e61457ad077ec0ffbb5415528456517ddb/nerfstudio/viewer/app/package.json#L4
This prevents issues with backwards compatability.

**Review by tancik — APPROVED** [review]
LGTM, thanks for adding

**Pull request merged**